images/in-repo-config-plugin/Dockerfile

```
@@ -0,0 +1,3 @@
+FROM quay.io/centos/centos:stream8
+ADD in-repo-config-plugin /usr/bin/in-repo-config-plugin
```

Hadolint (2.14.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)